declarations/Expedia.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "Expedia",
+  "documents": {
+    "Terms of Service": {
```

The document describes how “Expedia Group”, the `service provider`, provides “various tools and services to enter, upload or request changes to content or other information that is associated with or displayed on the Websites (as defined below) regarding a hotel, resort, inn, or other accommodation (“Property”) or a Property’s products and/or services”.
The recipients are clearly Property managers, since the audience has to “possess the legal authority to bind the Property to the terms of this Partner Central Agreement”. In the context of Expedia, a marketplace for property reservation, they are thus `commercial partners` using Expedia for `business or commercial usage`, not as renters themselves.

I thus conclude that this document is actually:
```suggestion
    "Commercial Terms": {
```